tests/presentations/indexable-presentation/og-title-test.php

```diff
@@ -10,6 +10,8 @@
  * @coversDefaultClass \Yoast\WP\Free\Presentations\Indexable_Presentation
  *
  * @group presentations
+ * @group og
```

Comment:
This introduces a third variation of our groups. There is already `opengraph` and `open-graph`.

Reply:
Changed all occurrences to `opengraph`